src/main/java/com/Ultramega/CentrifugeTiersReproduced/blocks/TieredCentrifuge.java

```diff
@@ -53,4 +61,72 @@ private BlockEntityType getBlockEntity() {
+                }
+                case NUCLEAR -> {
+                    outputMultiplier = CentrifugeTiersReproducedConfig.NUCLEAR_CENTRIFUGE_OUTPUT_MULTIPLIER.get();
+                    speedMultiplier = CentrifugeTiersReproducedConfig.NUCLEAR_CENTRIFUGE_SPEED.get();
+                    additionalCombSlots = 2;
+                    maxStackSize = CentrifugeTiersReproducedConfig.NUCLEAR_CENTRIFUGE_ITEM_MAX_STACK_SIZE.get();
+                    energyFluidCapacity = CentrifugeTiersReproducedConfig.NUCLEAR_CENTRIFUGE_ENERGY_CAPACITY.get();
+                }
+                case COSMIC -> {
+                    outputMultiplier = CentrifugeTiersReproducedConfig.COSMIC_CENTRIFUGE_OUTPUT_MULTIPLIER.get();
+                    speedMultiplier = CentrifugeTiersReproducedConfig.COSMIC_CENTRIFUGE_SPEED.get();
+                    additionalCombSlots = 3;
+                    maxStackSize = CentrifugeTiersReproducedConfig.COSMIC_CENTRIFUGE_ITEM_MAX_STACK_SIZE.get();
+                    energyFluidCapacity = CentrifugeTiersReproducedConfig.COSMIC_CENTRIFUGE_ENERGY_CAPACITY.get();
+                }
+                case CREATIVE -> {
+                    outputMultiplier = CentrifugeTiersReproducedConfig.CREATIVE_CENTRIFUGE_OUTPUT_MULTIPLIER.get();
+                    speedMultiplier = 0;
+                    additionalCombSlots = 3;
+                    maxStackSize = CentrifugeTiersReproducedConfig.CREATIVE_CENTRIFUGE_ITEM_MAX_STACK_SIZE.get();
+                    energyFluidCapacity = CentrifugeTiersReproducedConfig.CREATIVE_CENTRIFUGE_FLUID_CAPACITY.get();
+                }
+                default -> {
+                    outputMultiplier = 0;
+                    speedMultiplier = 0;
+                    additionalCombSlots = 0;
+                    maxStackSize = 0;
+                    energyFluidCapacity = 0;
+                }
+            }
+
+            String toolTipText;
+            if(tier == CentrifugeTiers.HIGH_END) {
+                toolTipText = String.format("%sx the output\n%sx faster than Heated Centrifuge\n%s additional honeycomb slot\nItem Max Stack Size: %s\nEnergy/Fluid Capacity: %s",
+                        outputMultiplier, speedMultiplier, additionalCombSlots, maxStackSize, energyFluidCapacity);
+            } else if (tier == CentrifugeTiers.CREATIVE) {
+                toolTipText = String.format("%sx the output\n1 Tick per Block\n%s additional honeycomb slots\nItem Max Stack Size: %s\nEnergy/Fluid Capacity: %s",
+                        outputMultiplier, additionalCombSlots, maxStackSize, energyFluidCapacity);
+            } else {
+                toolTipText = String.format("%sx the output\n%sx faster than Heated Centrifuge\n%s additional honeycomb slots\nItem Max Stack Size: %s\nEnergy/Fluid Capacity: %s",
+                        outputMultiplier, speedMultiplier, additionalCombSlots, maxStackSize, energyFluidCapacity);
+            }
```

Comment:
Don't use /n instead look at for example https://github.com/starforcraft/Centrifuge-Tiers/blob/f9f8d127279aa5210b71f4d99d170d854de17b32/src/main/java/com/YTrollman/CentrifugeTiers/block/CentrifugeControllerBlockTier3.java#L53